app/routes/slack_bot.py

```diff
@@ -0,0 +1,21 @@
+
+import os #newly added for wave4
+import requests #newly added for wave4
+from dotenv import load_dotenv
+
+load_dotenv()
+
+path = 'https://slack.com/api/chat.postMessage'
+SLACK_TOKEN = os.environ.get("SLACK_TOKEN")
+channel = "task_notifications"
```

Comment:
I would make all of these fully capitalized because they're essentially constants.